src/components/patterns/sul-accordion.tsx

```diff
@@ -48,7 +49,7 @@ const SulAccordion = ({button, children, headingLevel = "h2", ...props}: Props)
       aria-labelledby={`${id}-button`}
       className={twMerge("relative w-full", props.className)}
     >
-      <Heading className="type-0 mb-0 font-sans font-semibold">
+      <Heading className="type-0 mb-0 font-sans font-semibold" {...headingProps}>
```

Comment:
`{...props.headingProps}` was throwing a DOM error so I tweaked the props.

Reply:
oh thank you.